lib/dor/services/metadata_service.rb

```diff
@@ -69,13 +70,21 @@ def handler_for(prefix)
 
       private
 
+      # def handlers
+      #   @handlers ||= {}
+      # end
       def handlers
-        @handlers ||= {}
+        @handlers ||= {}.tap do |handlers|
```

Comment:
This inner variable is never used?

Reply:
Would you mind calling the variable `handlers` something different than the method name?